splitio/push/manager.py

```diff
@@ -143,7 +143,7 @@ def _trigger_connection_flow(self):
             self._feedback_loop.put(Status.PUSH_RETRYABLE_ERROR)
             return
 
-        if not token.push_enabled:
+        if token is None or not token.push_enabled:
```

Comment:
I think we should change the logic of Token dto and instead of returning this
`    return None if push_enabled is None else Token(push_enabled, token, json.loads(decoded_token['x-ably-capability']), decoded_token['exp'], decoded_token['iat'])
`
return Token(False, whatever, whatever, whatever, whatever) is still a valid Token dto without Token payload. Then, check push_enabled instead of None

Reply:
Will do